src/modules/components/proposal/proposalActionSimulationStructure/proposalActionSimulationStructure.tsx

```diff
@@ -0,0 +1,192 @@
+import classNames from 'classnames';
+import { DateTime } from 'luxon';
+import { useMemo } from 'react';
+import {
+    AvatarIcon,
+    Button,
+    DataList,
+    DateFormat,
+    DefinitionList,
+    formatterUtils,
+    IconType,
+    Spinner,
+} from '../../../../core';
+import { useGukModulesContext } from '../../gukModulesProvider';
+
+export interface IProposalActionSimulationStructureProps {
+    /**
+     * Total number of actions in the proposal.
+     */
+    totalActions: number;
+    /**
+     * Last simulation date.
+     */
+    lastSimulation?: DateTime;
+    /**
+     * Whether simulation is currently running.
+     */
+    isSimulating?: boolean;
+    /**
+     * Whether the proposal can be simulated.
+     */
+    isSimulatable?: boolean;
```

Comment:
We could reuse the `isLoading` / `isEnabled` pattern here 
```suggestion
    isLoading?: boolean;
    /**
     * Whether the proposal can be simulated.
     */
    isEnabled?: boolean;
```